lib/node.js

```diff
@@ -55,6 +56,28 @@ var Node = React.createClass({
 
     return null;
   },
+  
+  renderInner(){
+    if( this.isRoot() && ! this.props.shouldRenderRootNode ){
+      return null;
+    }
+    
+    var tree = this.props.tree;
+    var index = this.props.index;
+    var node = index.node;
+
+    return(
+        <div className="inner" ref="inner" onMouseDown={this.handleMouseDown}>
+          {this.renderCollapse()}
+          {tree.renderNode(node)}
+        </div>
+     )
+  },
+
+  isRoot(){
+    var index = this.props.index;
+    return index.id === 1 ? true : false;
```

Comment:
This line could be written like this:
`return index.id === 1`
with same result and less coding :smiley: